example/pythonlib/basic/1-simple/foo/src/foo.py

```diff
@@ -1,2 +1,22 @@
-import numpy as np
-data = np.array([10, 20, 30, 40, 50])
\ No newline at end of file
+#!/usr/bin/python3
```

Comment:
We don't need the shebang line for this script do we? I assume the `run` task will use the correct venv python binary automatically

Reply:
yes it will work! 

Fixed!